tests/rgw/sanity_rgw_multisite.py

```diff
@@ -75,11 +74,27 @@ def run(**kw):
             user_details_file, test_site_node, copy_user_to_site_node, user_details_file
         )
 
-    for site in verify_io_on_sites:
-        # todo: need to implement verify io section
-        log.info(f"verification IO on {site}")
-        raise NotImplementedError("verify IO still pending")
-
+    verify_io_on_site = clusters.get(config.get("verify-io-on-site"))
```

Comment:
There is an indentation error in the below lines... you could resolve them by running `black tests/rgw/sanity_rgw_multisite.py` from the command line.